workshop/exercise-1/README.md

```diff
@@ -6,92 +6,97 @@ You must already have a [cluster created](https://console.bluemix.net/docs/conta
 
 1. Install the IBM Cloud [command line interface](https://console.bluemix.net/docs/cli/reference/bluemix_cli/get_started.html#getting-started).
 
-2.  Log in to the IBM Cloud CLI. If you have a federated account, include the `[--sso]` flag.   
-    
-    ```bash
-    ibmcloud login [--sso]
-    ```
-
-    or if you have an api key, use:
-    ```bash
-    ibmcloud login --apikey {apikey}
-    ```
-    
+2.  Log in to the IBM Cloud CLI. If you have a federated account, include the `[--sso]` flag.
+
+```shell
+ibmcloud login [--sso]
+```
+
+or if you have an api key, use:
+
+```shell
+ibmcloud login --apikey <apikey>
+```
+
 3.  Install the IBM Cloud Kubernetes Service plug-in.
-    ```bash
-    ibmcloud plugin install container-service -r Bluemix
-    ```
+
+```shell
+ibmcloud plugin install container-service -r Bluemix
+```
 
 4. To verify that the plug-in is installed properly, run `ibmcloud plugin list`. The Container Service plug-in is displayed in the results as `container-service`.
 
-5.  Initialize the Container Service plug-in and point the endpoint to your region. For example when prompted, enter `5` for `us-east`.   
-    ```bash
-    $ ibmcloud cs region-set
-    Choose a region:
-    1. ap-north
-    2. ap-south
-    3. eu-central
-    4. uk-south
-    5. us-east
-    6. us-south
-    Enter a number> 5
-    ```
-    
-6. Install the Kubernetes CLI. Go to the [Kubernetes page](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl) to install the CLI and follow the steps.
+5.  Initialize the Container Service plug-in and point the endpoint to your region. For example when prompted, enter `5` for `us-east`.
 
+Example:
+
+```shell
+$ ibmcloud cs region-set
```

Comment:
this is the only spot where we show the prompt ($) - I actually prefer to show it so people know which lines are the ones they type vs output, but either way we should be consistent across all of the files in the repo